client/tracing/src/logging/directives.rs

```diff
@@ -81,11 +81,12 @@ pub fn reload_filter() -> Result<(), String> {
 			}
 		}
 	}
-	env_filter = env_filter.add_directive(
-		"sc_tracing=trace"
-			.parse()
-			.expect("provided directive is valid"),
-	);
```

Comment:
I don't remember why we added this?

Reply:
Nevermind it was probably a copy-paste from the log initialization when we run with tracing enabled.

Reply:
We have it here to have support for this tracing stuff. This whole stuff is a little bit confusing and not used by anyone. If someone wants to use it, they need to start the node with tracing enabled.